gulpfile.js

```diff
@@ -27,7 +31,7 @@ require('gulp-task-loader')({
 // Define task for each theme, locale, lang, processing type etc.
 // Gulp can't run same task in parallel, so we need different names
 Object.keys(config.themes).forEach(name => {
-  var theme = config.themes[name];
+  const theme = config.themes[name];
   theme.locale.forEach(locale => {
     gulp.task(
       theme.lang + ':' + name + ':' + locale,
```

Comment:
> I think if we have a task to setup configuration, throwing errors while themes.json doesn't exist might not be so useful anymore.

I agree, only reason I added it is for when the config file can't be found. Otherwise things like `gulp styles` will run fine but won't do anything ofcourse

> To be honest I don't like idea of get-themes. IMO it's pointless to keep this as separate helper.

The theme reading logic is used about 5 times, that's why I seperated it.

> My goal is to keep things simple, but in this case simplifying tools for noobs (sorry, but it's super easy to copy / create configuration in right place) are adding complexity to whole project.

I agree that we should keep things simple. The changes I made aren't so complex, it's a change in function indeed. But the only real change is that `config-loader` now has an optional parameter to disregard the file not being there. I also added a themes helper which replaces a function that is used 5 times and improved it with a check to make sure the provided theme in `--theme <theme name>` is defined.